src/hash.cr

```diff
@@ -1855,6 +1855,7 @@ class Hash(K, V)
   # h = {1 => 'a', 2 => 'b', 3 => 'c'}
   # h.to_a # => [{1, 'a'}, {2, 'b'}, {3, 'c'}]
   # ```
+  # The order of the array follows the order the keys were inserted in the Hash.
```

Comment:
The fact that the ordered is preserved can be put on the main method description above.

Reply:
Yeah it does seem weird at the bottom.  For whatever reason all the other methods did it this way so I just followed suit...

Reply:
I think this is a coincidence due to the implementation, but this isn't true every time.

Reply:
It's not a coincidence.

Reply:
Ok :+1: